create_jenkins_job.py

```diff
@@ -120,7 +120,7 @@ def main(argv=None):
         'enable_coverage_default': 'false',
         'dont_notify_every_unstable_build': 'false',
         'build_timeout_mins': 0,
-        'ubuntu_distro': 'jammy',
+        'ubuntu_distro': 'noble',
```

Comment:
We don't want to switch the default to `noble` yet, so let's leave this as `jammy` for now.

Reply:
Changed back in https://github.com/ros2/ci/pull/738/commits/89f939db1cb7a899c0e018ba85ffb158950b217b.